src/installation.md

```diff
@@ -51,4 +51,5 @@ sudo reboot
 
 ## Step 8: Launched Sugar
 
-Once the Raspberry Pi reboots, Sugar should start automatically. If not you may have to follow up with troubleshoots.
\ No newline at end of file
+
+After rebooting the Raspberry Pi, Sugar should start automatically. Remember to verify whether the Sugar Desktop Environment is installed and to disable auto-login in the RPi-config settings. If you encounter any difficulties, you may need to proceed with troubleshooting
```

Comment:
Troubleshoot has to be done either with the vm , or any hardware or any system that has been used or it can be related with sugar , troubleshoot here generally means that seeing if there are any error/problems by either going again through doc or going through setup , i think troubleshoot word should be self explanatory because there can be different types error/os/vm.
if there is any other way to rewrite this than that can be added.

Reply:
"After rebooting the Raspberry Pi, Sugar should start automatically. Remember to verify whether the Sugar Desktop Environment is installed and to disable auto-login in the RPi-config settings. If you encounter any difficulties, you may need to proceed with further steps to resolve the issue"

Reply:
will this work or should i remove encounter any difficulties line altogether.

Reply:
I think that unless you are going to explain how to trouble-shoot, you should leave it out.